march_hardware/include/march_hardware/IMotionCube.h

```diff
@@ -96,13 +100,15 @@ class IMotionCube : public Slave
   void mapMisoPDOs();
   void mapMosiPDOs();
   void writeInitialSettings(uint8_t cycle_time);
+  uint16_t computeSWCheckSum(int& start_address, int& end_address);
```

Comment:
I think it would be useful if you add documentation on this method to describe what it returns and what is expected from the two int reference parameters.

Reply:
I will provide a docstring above the function for clarification.